terminal-to-che.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pattern="${1:-workspace}"
+
+kubectl exec -it $(kubectl get pod | grep -o "${pattern}\S\+" | head -n 1) -- bash
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)